assignment5.js

```diff
@@ -0,0 +1,30 @@
+var fs = require('fs');
+
+var fileReader = (fileName) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(fileName, 'utf8', (error, data) => {
+      console.log(data); 
```

Comment:
Consider the case of error. If there is an error than, then data will have value `undefined` and will print unnecessarily. You need to rearrange your some of the lines in this callback function.